### PR TITLE
Add methods about invitations in auth-server in Litmus

### PR DESCRIPTION
**Pull request by inpyu**
I created methods that wraps APIs related to invite
And, I also implemented test code and Docs for these APIs.

### CODE REVIEW

**Comment by inpyu** [comment]
@suyeon-jung-dev @jemlog @seedspirit Can you please take a look?

**Comment by seedspirit** [comment]
LGTM 👍

File: src/test/java/InviteTest.java [hunk 1]
```diff
@@ -0,0 +1,209 @@
...
+    }
+
+    @Test
+    public void testAcceptInvitation() throws IOException, LitmusApiException {
+        System.out.println("projectId: " + projectId + " userId: " + userId);
+        createInvite(projectId, userId);
+        AcceptInvitationRequest request = AcceptInvitationRequest.builder()
+                .projectId(projectId)
+                .userId(userId)
+                .build();
```
Comment (jemlog) [inline src/test/java/InviteTest.java:135]: Can you remove console print method? 🙂

File: src/test/java/InviteTest.java [hunk 2]
```diff
@@ -0,0 +1,209 @@
...
+
+        List<String> tags = Arrays.asList("tag1", "tag2");
+        String projectName = generateUniqueProjectName("Temporary Project");
+        CreateProjectRequest projectRequest = CreateProjectRequest.builder()
+                .projectName(projectName)
+                .description("testDescription")
+                .tags(tags)
+                .build();
+
+        ProjectResponse projectResponse = litmusClient.createProject(projectRequest);
```
Comment (jemlog) [inline src/test/java/InviteTest.java:198]: I think we need to remove created project after test finished for independent tests.

**Review by jemlog — APPROVED** [review]
Nice work @inpyu !
Your documentation is so helpful 👍

File: src/main/java/io/litmuschaos/request/AcceptInvitationRequest.java [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+        public Builder userId(String userId) {
+            this.userId = userId;
+            return this;
+        }
+
+        public AcceptInvitationRequest build() {
+            return new AcceptInvitationRequest(this.projectId, this.userId);
+        }
+    }
```
Comment (suyeon-jung-dev) [inline src/main/java/io/litmuschaos/request/AcceptInvitationRequest.java:34]: Could you implements Builder class in `io.litmuschaos.util.Builder`?

File: src/main/java/io/litmuschaos/request/AcceptInvitationRequest.java [hunk 4]
```diff
@@ -0,0 +1,35 @@
+package io.litmuschaos.request;
+
+public class AcceptInvitationRequest {
+
+    private String projectId;
+    private String userId;
```
Comment (suyeon-jung-dev) [inline src/main/java/io/litmuschaos/request/AcceptInvitationRequest.java:6]: Could you consider making these fields `final` to ensure immutability?

```java
private final String projectId;
private final String userId;

**Review by suyeon-jung-dev — CHANGES_REQUESTED** [review]
Great work @inpyu ! 👍👍 I’ve left comments for one class, but could you review and apply similar improvements across the other classes as well?

File: src/test/java/InviteTest.java (hunk 1 above)
Comment (inpyu) [reply src/test/java/InviteTest.java:135]: Oh, I didn't check it. thanks!

**Comment by inpyu** [comment]
I've updated the Invite Client Code. thanks for your kind example!  Could you please review it? @suyeon-jung-dev @jemlog

**Comment by jemlog** [comment]
👍

**Pull request merged**